Packs/JARM/Integrations/JARM/JARM.yml

```diff
@@ -0,0 +1,56 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: JARM
+  version: -1
+configuration:
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'Active TLS fingerprinting using JARM'
+display: JARM
+name: JARM
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: 'FQDN or IP address to fingerprint. Also supports [https://fqdn:port]
+        format'
+      isArray: false
+      name: host
+      required: true
+      secret: false
+    - default: false
+      description: 'Port to fingerprint. If provided overrides the port specified
+        in the host parameter'
+      isArray: false
+      name: port
+      required: false
+      defaultValue: 443
+      secret: false
+    deprecated: false
+    description: 'Calculate JARM fingerprint by scanning host with multiple TLS packets.'
+    execution: false
+    name: jarm-fingerprint
+    outputs:
+    - contextPath: JARM.FQDN
+      description: 'FQDN of the host.'
+      type: String
+    - contextPath: JARM.IP
+      description: 'IP Address of the host.'
+      type: String
+    - contextPath: JARM.Port
+      description: 'TCP port'
+      type: Number
+    - contextPath: JARM.Fingerprint
+      description: 'JARM fingerprint of the host.'
+      type: String
```

Comment:
Please add the DbotScore here
You can take a look here https://xsoar.pan.dev/docs/integrations/dbot

Reply:
done